resources/hu/cemu.po

```diff
@@ -1,1333 +1,1780 @@
+msgstr "Játékprofil szerkesztése"
+
+#: GameProfileWindow.cpp:32 GameProfileWindow.cpp:80 GeneralSettings2.cpp:158
+#: GeneralSettings2.cpp:166 GeneralSettings2.cpp:300
+msgid "General"
+msgstr "Általános"
+
+#: GameProfileWindow.cpp:35
+msgid "Load shared libraries"
+msgstr "Megosztott könyvtárak betöltése"
+
+#: GameProfileWindow.cpp:36
+msgid ""
+"EXPERT OPTION\n"
+"This option will load libraries from the cafeLibs directory"
+msgstr ""
+"CSAK HALADÓKNAK\vEz a parancs a cafeLibs mappából fog könyvtárakat betölteni"
+
+#: GameProfileWindow.cpp:43
+msgid "CPU"
+msgstr "CPU"
+
+#: GameProfileWindow.cpp:50
+msgid "Mode"
+msgstr "Mód"
+
+#: GameProfileWindow.cpp:52
+msgid "Singlecore-Interpreter"
+msgstr "Egymagos interpreter"
+
+#: GameProfileWindow.cpp:52
+msgid "Singlecore-Recompiler"
+msgstr "Egymagos recompiler"
+
+#: GameProfileWindow.cpp:52
+msgid "Dualcore-Recompiler"
+msgstr "Kétmagos recompiler"
+
+#: GameProfileWindow.cpp:52
+msgid "Triplecore-Recompiler"
+msgstr "Hárommagos recompiler"
+
+#: GameProfileWindow.cpp:55
+msgid "Set the CPU emulation mode"
+msgstr "Állítsd be a CPU-emulációs módot"
+
+#: GameProfileWindow.cpp:60
+msgid "Thread quantum"
+msgstr "Thread quantum"
+
+#: GameProfileWindow.cpp:65
+msgid ""
+"EXPERT OPTION\n"
+"Set the maximum thread slice runtime (in virtual cycles)"
+msgstr ""
+"CSAK HALADÓKNAK\n"
+"Állítsd be a legmagasabb szál slice futásidejét (virtuális ciklusokban)"
+
+#: GameProfileWindow.cpp:68
+msgid "cycles"
+msgstr "ciklus"
+
+#: GameProfileWindow.cpp:90
+msgid "Extended texture readback"
+msgstr "Kiterjesztett textúra-visszaolvasás"
+
+#: GameProfileWindow.cpp:91
+msgid ""
+"Improves emulation accuracy of CPU to GPU memory access at the cost of "
+"performance. Required for some games."
+msgstr ""
+"Segít a CPU-GPU memóriahozzáférés emulációs pontosságán a teljesítmény "
+"feláldozásával. Néhány játékhoz kötelező."
+
+#: GameProfileWindow.cpp:98 GeneralSettings2.cpp:190
+msgid "Precompiled shaders"
+msgstr "Előre összeállított shaderek"
+
+#: GameProfileWindow.cpp:99 GeneralSettings2.cpp:191
+msgid "auto"
+msgstr "automatikus"
+
+#: GameProfileWindow.cpp:99 GeneralSettings2.cpp:191
+msgid "enable"
+msgstr "engedélyezés"
+
+#: GameProfileWindow.cpp:99 GeneralSettings2.cpp:191
+msgid "disable"
+msgstr "letiltás"
+
+#: GameProfileWindow.cpp:102 GeneralSettings2.cpp:195
+msgid ""
+"Precompiled shaders can speed up the load time on the shader loading "
+"screen.\n"
+"Auto will enable it for AMD/Intel but disable it for NVIDIA GPUs as a "
+"workaround for a driver bug.\n"
+"\n"
+"Recommended: Auto"
+msgstr ""
+"Az előre összeállított shaderek felgyorsíthatják a betöltés idejét a "
+"shaderbetöltési képernyőn.\n"
+"Az automatikus üzemmód engedélyezi AMD/Intel GPU-k esetén, viszont letiltja "
+"az NVIDIA GPU-knál egy illesztőprogramhiba kiküszöbölése érdekében.\n"
+"\n"
+"Javasolt érték: Automatikus"
+
+#: GameProfileWindow.cpp:105
+msgid "Shader mul accuracy"
+msgstr "Shader mul pontosság"

```

Comment:
In the English string "mul" is a shorthand for "multiplication". Is that immediately clear for an Hungarian reader? If not, use the Hungarian term for "multiplication".